src/components/home/modals/Send.vue

```diff
@@ -634,6 +659,31 @@ export default {
                 } else {
                     this.coldPageId++
                 }
+                if (this.from3rdParty) {
+                    if (this.paymentRedirect.hasOwnProperty('callback')) {
```

Comment:
The callback should start call once the transaction is sent, not after close the modal.